### PR TITLE
Remove more nullable+embed options

### PR DESCRIPTION
**Pull request by htdvisser**
#### Summary

This removes more gogoproto.nullable and gogoproto.embed options from `application.proto`, `client.proto`, `organization.proto` and `user.proto`.

Refs #2798

#### Testing

Existing unit tests

##### Regressions

Same story as in the previous pull requests

#### Notes for Reviewers

An additional change I made while I was editing these files was renaming `APIKey` variable names to `apiKey`.

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [ ] Documentation: Relevant documentation is added or updated.
- [ ] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: pkg/ttnpb/application_interfaces.go [hunk 1]
```diff
@@ -100,6 +116,10 @@ func (m *GetApplicationRequest) ExtractRequestFields(dst map[string]interface{})
 	m.GetApplicationIds().ExtractRequestFields(dst)
 }
 
+func (m *UpdateApplicationRequest) ExtractRequestFields(dst map[string]interface{}) {
```
Comment (adriansmares) [inline pkg/ttnpb/application_interfaces.go:119]: I don't see `ExtractRequestFields` for `CreateApplicationRequest` - is this intentional ?

File: pkg/ttnpb/client_interfaces.go [hunk 2]
```diff
@@ -68,6 +84,10 @@ func (m *GetClientRequest) ExtractRequestFields(dst map[string]interface{}) {
 	m.GetClientIds().ExtractRequestFields(dst)
 }
 
+func (m *UpdateClientRequest) ExtractRequestFields(dst map[string]interface{}) {
```
Comment (adriansmares) [inline pkg/ttnpb/client_interfaces.go:87]: Same here for `CreateClientRequest`.

**Review by adriansmares — COMMENTED** [review]
I'm missing on why the `Create*` requests don't have an `ExtractRequestFields` entry.

File: pkg/ttnpb/application_interfaces.go (hunk 1 above)
Comment (htdvisser) [reply pkg/ttnpb/application_interfaces.go:119]: Already exists in `pkg/ttnpb/logfields.go`.

File: pkg/ttnpb/client_interfaces.go (hunk 2 above)
Comment (htdvisser) [reply pkg/ttnpb/client_interfaces.go:87]: Already exists in `pkg/ttnpb/logfields.go`.

**Review by adriansmares — APPROVED** [review]

**Pull request merged**